gex/Controllers/Api/UserApiController.cs

```diff
@@ -318,6 +318,63 @@ public async Task<ApiResponse<List<UserSearchResult>>> Search(string search,
             return ApiOk(users);
         }
 
+        /// <summary>
+        ///     search for multiple users based on a list of search terms. A max of 50 terms can be searched at once.
+        /// </summary>
+        /// <param name="users">list of search terms. each term must be at least 3 characters long</param>
+        /// <param name="includeSkill">if <see cref="UserSearchResult.Skill"/> will be populated. defaults to false</param>
+        /// <param name="searchPreviousNames">will previous names be searched against as well? defaults to false</param>
+        /// <param name="cancel">cancellation token</param>
+        /// <response code="200">
+        ///     the response will contain a list of <see cref="UserSearchResult"/>s that match any of the search terms
+        /// </response>
+        /// <response code="400">
+        ///     one or more search terms are not at least 3 characters long, or no search terms provided
+        /// </response>
+        [HttpPost("search/batch")]
+        public async Task<ApiResponse<List<UserSearchResult>>> SearchMultipleUsers(
```

Comment:
this can be a GET, no reason for a POST here. just have `users` be a `[FromQuery]`